lib/gui/models/store.js

```diff
@@ -46,6 +48,11 @@ const DEFAULT_STATE = Immutable.fromJS({
     unmountOnSuccess: true,
     validateWriteOnSuccess: true,
     sleepUpdateCheck: false,
+
+    // Only set this by default if the installed
+    // version is not a production version
+    includeUnstableUpdateChannel: release.getReleaseType(packageJSON.version) !== release.RELEASE_TYPE.PRODUCTION,
```

Comment:
Doh! You've even muddled up stable/unstable and production/snapshot yourself! :-/
I believe this should be checking the result of `.isStableRelease` rather than the result of `.getReleaseType`.